### PR TITLE
BZ2218185: Monitoring section with Runbooks

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.14

Issue: [CNV-30368](https://issues.redhat.com/browse/CNV-30368), [BZ#2218185](https://bugzilla.redhat.com/show_bug.cgi?id=2218185)

Link to docs preview:

- [Monitoring overview](https://61902--docspreview.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-monitoring-overview.html)
- [OpenShift cluster checking framework](https://61902--docspreview.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-running-cluster-checkups.html)
- [Prometheus queries for virtual resources](https://61902--docspreview.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-prometheus-queries.html#virt-prometheus-queries)
- [VM custom metrics](https://61902--docspreview.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-exposing-custom-metrics-for-vms.html)
- [VM health checks](https://61902--docspreview.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-monitoring-vm-health.html)
- [Runbooks](https://61902--docspreview.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-runbooks.html)

QE review: NA
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by fabiand** [comment]
I did not check everything, but yep - that's looking good!

File: virt/monitoring/virt-prometheus-queries.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ Use the {product-title} monitoring dashboard to query virtualization metrics.
 [id="prerequisites_{context}"]
 == Prerequisites
 
-* To use the vCPU metric, the `schedstats=enable` kernel argument must be applied to the `MachineConfig` object. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler. For more information, see xref:../../../post_installation_configuration/machine-configuration-tasks.html#nodes-nodes-kernel-arguments_post-install-machine-configuration-tasks[Adding kernel arguments to nodes].
+* To use the vCPU metric, the `schedstats=enable` kernel argument must be applied to the `MachineConfig` object. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler. For more information, see xref:../../post_installation_configuration/machine-configuration-tasks.html#nodes-nodes-kernel-arguments_post-install-machine-configuration-tasks[Adding kernel arguments to nodes].
```
Comment (michaelryanpeter) [inline virt/monitoring/virt-prometheus-queries.adoc:17]:
```suggestion
* To use the vCPU metric, the `schedstats=enable` kernel argument must be applied to the `MachineConfig` object. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler. For more information, see xref:../../post_installation_configuration/machine-configuration-tasks.adoc#nodes-nodes-kernel-arguments_post-install-machine-configuration-tasks[Adding kernel arguments to nodes].
```

Comment (michaelryanpeter) [reply virt/monitoring/virt-prometheus-queries.adoc:17]: Nit that was in the original content :)

**Review by michaelryanpeter — COMMENTED** [review]
Just one small nit, which was in the original content. Great job!

**Pull request merged**